src/Abstractions/src/Asp.Versioning.Abstractions/ISunsetPolicyManager.cs

```diff
@@ -5,18 +5,5 @@ namespace Asp.Versioning;
 /// <summary>
 /// Defines the behavior of an API version sunset policy manager.
 /// </summary>
-public interface ISunsetPolicyManager
-{
-    /// <summary>
-    /// Returns the sunset policy for the specified API and version.
-    /// </summary>
-    /// <param name="name">The name of the API.</param>
-    /// <param name="apiVersion">The API version to get the policy for.</param>
-    /// <param name="sunsetPolicy">The applicable <see cref="SunsetPolicy">sunset policy</see>, if any.</param>
-    /// <returns>True if the <paramref name="sunsetPolicy">sunset policy</paramref> was retrieved; otherwise, false.</returns>
-    /// <remarks>If <paramref name="name"/> is <c>null</c>, it is assumed the caller intends to match any sunset
-    /// policy for the specified <paramref name="apiVersion">API version</paramref>. If
-    /// <paramref name="apiVersion">API version</paramref> is <c>null</c>, it is assumed the caller intends to match
-    /// any sunset policy for the specified <paramref name="name"/>.</remarks>
-    bool TryGetPolicy( string? name, ApiVersion? apiVersion, [MaybeNullWhen( false )] out SunsetPolicy sunsetPolicy );
-}
\ No newline at end of file
+public interface ISunsetPolicyManager : IPolicyManager<SunsetPolicy>
```

Comment:
Is this worth keeping? I'm sure you did this to avoid the breaking change. Since we're at a major version LTS release, I'd be OK with removing `ISunsetPolicyManager`. It's just a backward-compatible, marker interface now. If this were to be introduced as a breaking change, now is probably the time. Historically, few - if any - ever dive down to this level of customization so as long as everything else is source compatible, it's doubtful to be problematic.

I'm also considering doing at least one preview release, which _might_ surface if anyone would be upset by that.